test/model/message_test.dart

```diff
@@ -628,13 +628,13 @@ void main() {
           eg.submessage(senderId: effectiveMessageSender.userId,
             content: eg.pollWidgetData(
               question: question ?? 'example question',
-              options: (options != null)
-                ? options.map((e) => e.text).toList()
+              options: (optionVoterIdsPairs != null)
+                ? optionVoterIdsPairs.map((e) => e.$1).toList()
```

Comment:
Oh, odd, I guess `$1` refers to the first element of the tuple? So it's not zero-indexed.